package.json

```diff
@@ -647,6 +648,7 @@
     "style-loader": "^0.21.0",
     "stylelint": "^13.6.1",
     "superstruct": "^1.0.3",
+    "tailwindcss": "^3.4.1",
```

Comment:
Adding tailwind css dependency - v4 has been released but most projects using tailwind in MM are on v3